README.rst

```diff
@@ -57,15 +57,82 @@ Installation
 ============
 We support Linux and MacOS (please let us know if you are successful in installing on
 Windows!). On these systems, the simplest way to get `21cmFAST` is by using
-`<https://www.anaconda.com/>conda`_::
+`conda<https://www.anaconda.com/>`_::
 
     conda install -c conda-forge 21cmFAST
 
 `21cmFAST` is also available on PyPI, so that `pip install 21cmFAST` also works. However,
 it depends on some external (non-python) libraries that may not be present, and so this
 method is discouraged unless absolutely necessary. If using `pip` to install `21cmFAST`
 (especially on MacOS), we thoroughly recommend reading the detailed
-`<https://21cmfast.readthedocs.io/en/latest/installation.html>installation instructions`_.
+`installation instructions<https://21cmfast.readthedocs.io/en/latest/installation.html>`_.
+
+Basic Usage
+===========
+``21cmFAST`` can be run both interactively and from the command line (CLI).
+
+Interactive
+-----------
+The most basic example of running a (very small) coeval simulation at a given redshift,
+and plotting an image of a slice through it::
+
+    >>> import py21cmfast as p21c
+    >>> coeval = p21c.run_coeval(
+    >>>     redshift=8.0,
+    >>>     user_params={'HII_DIM': 50, "USE_INTERPOLATION_TABLES": False}
+    >>> )
+    >>> p21c.plotting.coeval_sliceplot(coeval, kind='brightness_temp')
+
+The coeval object here has much more than just the ``brightness_temp`` field in it. You
+can plot the ``density`` field, ``velocity`` field or a number of other fields.
+To simulate a full lightcone::
+
+    >>> lc = p21c.run_lightcone(
+    >>>     redshift=8.0,
+    >>>     max_redshift=15.0,
```

Comment:
is the default to not include spin temperature fluctuations?  if not, then z=15 is too low.

Reply:
Yes, default is to not include spin temp.